### ISSUE TITLE
Active (focus) object change via Clipboard is ignored

### ISSUE THREAD

**Issue by emyoulation** [cdhorn/CardView#225]
Using the "Make the &lt;object> active" from the Clipboard (or Collections Clipboard) is ignored by all CardViews.  (Status Bar does not update to the new Active Object when in the CardViews either.) 

Switching to the built-in view modes have the Clipboard activated object. Switching back again has the updated active object.

Tested the Person Card, Event Card and Place Card.  The behavior was the same.
Version: 0.99.110

**Comment by cdhorn** [comment]
Yep.  Also ideally make the object active should switch the category if needed and it does not but that would require core code changes in the clipboard.
